### ISSUE TITLE
Security missconfiguration vulnerability 

### ISSUE THREAD

**Issue by HemalKp** [OWASP/crAPI#255]
Want to ask that is the crAPI have security missconfiguration vulnerability

**Comment by piyushroshan** [comment]
Yes it does. .env file leak, embedded harcoded jwks keys, default users.

**Comment by jeffrey-theog06** [comment]
Since it has security misconfig vuln like .env file leaks and so on as you listed.. is it solvable or can i give in a try to the fix?
@piyushroshan

**Comment by piyushroshan** [comment]
Sorry i did not understand you. This are intentional from a vulnerable app perspective

**Comment by jeffrey-theog06** [comment]
> Sorry i did not understand you. This are intentional from a vulnerable app perspective

Well i asked if i can work on the fix or no need?

**Comment by piyushroshan** [comment]
No fix is required as of now on this. Thank you regardless.
